src/bit-systems/media-loading.ts

```diff
@@ -19,6 +20,39 @@ import { loadAudio } from "../utils/load-audio";
 import { MediaType, mediaTypeName, resolveMediaInfo } from "../utils/media-utils";
 import { EntityID } from "../utils/networking-types";
 
+export const MediaContentBounds = (MediaLoaded as any).contentBounds as Map<EntityID, Vector3>;
```

Comment:
I'm not really sure if this alias pattern is readable because it is unclear `MediaContentBounds` data is associated with `MediaLoaded` component. At least, the name should clearly indicate the data is associated with `MediaLoaded` component.

Reply:
Right, I think I started using this for storing something different but ended being just a vector. I've refactored this to its own component.